spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -1262,7 +1262,7 @@ def name_id_version(format_urn)
             authn_context: ['http://idmanagement.gov/ns/assurance/loa/5'],
             authn_context_comparison: 'exact',
             request_signed: true,
-            requested_ial: 'http://idmanagement.gov/ns/assurance/loa/5',
+            requested_ial: 'none',
```

Comment:
[question] is this what we want? it feels like there is value in getting the invalid requested ial back

Reply:
also this indicates to me that behavior is changing with this PR, which should not be happening, i think (just made a note above about the LOA prefix)